### DISCUSSION TITLE
Removing lualine extensions

### DISCUSSION THREAD

**Discussion by trevarj** [LazyVim/LazyVim#2272] [Q&A] [answered]
I'm trying to get rid / edit this component on the lualine, but I believe that it's coming from the `extensions = { "neo-tree", "lazy" },` part of the LazyVim defaults.

![image](https://github.com/LazyVim/LazyVim/assets/5448324/36a85bd5-895e-4dae-94ee-4cd70c483c52)

Tried removing `opts.extensions` but my lua-fu is bad. Anyone know how?

**Comment by dpetka2001** [comment, marked as answer]
That component is defined [here](https://github.com/LazyVim/LazyVim/blob/879e29504d43e9f178d967ecc34d482f902e5a91/lua/lazyvim/plugins/ui.lua#L142-L143). As it is the first component in `lualine_c` you could do something like
```lua
  {
    "nvim-lualine/lualine.nvim",
    opts = function(_, opts)
      table.remove(opts.sections.lualine_c, 1)
    end,
  },
```
to remove it. All the lualine components are tables, so in this case you want to remove the first table of `lualine_c`.

**Reply by trevarj** [reply]
Awesome, thanks!!
